src/models/schema.ts

```diff
@@ -40,23 +40,14 @@ export class Schema extends Data implements IResourceBase {
   type!: SchemaType;
   // added separately from the response data, used for follow-on API calls
   schemaRegistryId!: Enforced<string>;
-  environmentId!: Enforced<string> | undefined;
+  environmentId!: EnvironmentId | undefined;
```

Comment:
Trying to set to `Enforced<EnvironmentId> | undefined` produced bizarre complaints. I wonder if is the ' | undefined' portion. Punting for now.